plugin/clone/include/clone_common.h

```diff
@@ -17,6 +17,7 @@
 #ifndef CLONE_COMMON_H
 #define CLONE_COMMON_H
 
+#include "clone.h"
```

Comment:
Why?

Reply:
to use Key_Values type declaration.

Reply:
The header file does not need to see the full declaration, forward declare it instead

Reply:
Right, in this case the "forward declaration" would be the duplication of the `using` directives. Please decide if that or an include looks better